charts/kube-prometheus-stack/Chart.yaml

```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 13.5.0
```

Comment:
thanks for the PR @verdel 

Can you pls rebase your fork to resolve the conflicts?

Reply:
:white_check_mark: